images/base/files/usr/local/bin/entrypoint

```diff
@@ -29,7 +29,12 @@ fi
 
 grep_allow_nomatch() {
   # grep exits 0 on match, 1 on no match, 2 on error
-  grep "$@" || [[ $? == 1 ]]
+  grep "$@"|| [[ $? == 1 ]]
```

Comment:
I guess this is an unintended change and reverted it in 3rd patch, please let me know if this is on purpose